english/bored/bored.md

```diff
@@ -9,3 +9,5 @@
 "Or should I go on an [ADVENTURE!](../eat-rainbows/rainbows.md)?"
 
 "Or maybe [escape to the wilderness](../escape-to-the-wilderness/wilderness.md)?"
+
+("Maybe I should go to a jazz club?")[../jazz-guitar.jazz.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.